bridges/opentracing/lib/opentelemetry/bridge/opentracing/scope_manager.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # A ScopeManager provides an API for interfacing with
+      # OpenTelemetry Tracers and Spans as OpenTracing objects
+      class ScopeManager
+        def initialize(tracer)
+          @tracer = tracer
+        end
+
+        # Activate the given span
+        #
+        # @param [Span] span An OpenTelemetry Span
+        # @return [Span]
+        def activate(span, finish_on_close: true)
+          Span.new(@tracer.with_span(span))
```

Comment:
This should:
1. Create a `Scope` with the given `span` and whether to `finish_on_close`.
1. Track the current active `Scope` in an instance variable.
1. Return the `Scope`.

After 👀 some more, it looks like the current `Context` related APIs in OpenTelemetry are slightly incompatible with the `Scope` API from OpenTracing - notably, OpenTelemetry `Context` is block-structured. It may be worth discussing whether that's the right thing to do. For now, though, I'd implement OpenTracing's `Scope` independently of OpenTelemetry's `Context`.